api/google-search/src/main/resources/amazon.json

```diff
@@ -0,0 +1,37 @@
+{
+  "total_results": 3990000000,
```

Comment:
`total_results` should be 6

Reply:
Updated all JSON(s) with the exact length of results.